### PR TITLE
Template

### PR DESCRIPTION
**Pull request by allabors**
Two languages blueprint stores new nfa in nfa catalog. (For e2e test please dont forget to run SQL script and to execute the new feature at least 5 times. It will increment the db-sequence the first 5 times and then you will be able to add a new nfa in nfa catalog succesfully).

### CODE REVIEW

File: nfafrontend/src/app/app.module.ts [hunk 1]
```diff
@@ -1,102 +1,108 @@
-import { BrowserModule } from '@angular/platform-browser';
```
Comment (jmnietsch) [inline nfafrontend/src/app/app.module.ts]: what happened here. Why are there 1000 chars changed, where basically nothing is different?

File: nfabackend/src/main/resources/ddl/NFADB.sql [hunk 2]
```diff
@@ -336,26 +336,20 @@ CREATE TABLE public.nfa
...
  RECHTLICHE_VERBINDLICHKEIT character varying(40),
- WERT real,
+ WERT character varying(200),
  FORMULIERUNG character varying(40),
- ERKLAERUNG character varying(500),
  REFERENZ character varying(40),
  REFERENZIERTE_PROJEKTE character varying(40),
- KRITIKALITAET character varying(40),    
-DOKUMENT character varying(40));
-INSERT INTO nfa VALUES (1,1,'Type:1','Bezeichnung:1','Verbindlichkeit:1','12.22','Formulierung:1','erklaerung:1','referenz:1','referenzierte_projekt:1','kritikalitaet', 'document1');
```
Comment (jmnietsch) [inline nfabackend/src/main/resources/ddl/NFADB.sql]: why did you remove these Values? Won't @HarishVijayamohan need them?

File: nfafrontend/src/app/app.module.ts (hunk 1 above)
Comment (allabors) [reply nfafrontend/src/app/app.module.ts]: Have no idea, I have resolved some conflicts inthis file with git tool in eclipse, only added some new imports

File: nfabackend/src/main/resources/ddl/NFADB.sql (hunk 2 above)
Comment (allabors) [reply nfabackend/src/main/resources/ddl/NFADB.sql]: I changed strucure of table and did not want to change all dummy data.

File: nfabackend/src/main/java/com/msg/nfabackend/entities/nfaCatalog.java [hunk 3]
```diff
@@ -1,17 +1,304 @@
...
+		}
+		/**
+		 * @param en the en to set
+		 */
+		public void setEn(BpPropertyTemplateNoConditionEn en) {
+			this.en = en;
+		}
+	}
+	
+	public static class BpPropertyTemplateNoConditionDe {
```
Comment (jmnietsch) [inline nfabackend/src/main/java/com/msg/nfabackend/entities/nfaCatalog.java]: What is the difference between this and the BpPropertyTemplateNoConditionEn? Why not use one Class twice???

File: nfabackend/src/main/java/com/msg/nfabackend/entities/nfaCatalog.java [hunk 4]
```diff
@@ -1,17 +1,304 @@
...
+		/**
+		 * @param verb the verb to set
+		 */
+		public void setVerb(String verb) {
+			this.verb = verb;
+		}
+		    
+	}
+	
+	public static class BpPropertyTemplateNoConditionEn {
```
Comment (jmnietsch) [inline nfabackend/src/main/java/com/msg/nfabackend/entities/nfaCatalog.java]: What is the difference between this and the BpPropertyTemplateNoConditionDe? Why not use one Class twice???

Comment (jmnietsch) [reply nfabackend/src/main/java/com/msg/nfabackend/entities/nfaCatalog.java]: Or create some inheritance structure.
